src/bp/core/services/migration/index.ts

```diff
@@ -14,9 +16,7 @@ import semver from 'semver'
 import stripAnsi from 'strip-ansi'
 import yn from 'yn'
 
-import { container } from '../../app.inversify'
-import { GhostService } from '../ghost/service'
-import { BotMigrationService } from './bot'
+import { container } from '../../app/app.inversify'
```

Comment:
... from 'core/app/inversify/app.inversify' ?

Reply:
Yeah I know, a bit of repetition, but in another PR i'm moving the 4 other inversify files. Maybe I could rename it "main.inversify" in another PR, but since they are all based on one another, i wont change it here

Reply:
My mistake, I meant to illustrate the fact that you were importing the container using a relative import instead of a alsolute one (`../../app/app.inversify` vs `core/app/inversify/app.inversify`).